contracts/dev/functions/migrations/FunctionsOracleMigration.sol

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I forgot why we need the migrations dir. Can't we just upgrade the contracts and merge into main afterwards?